packages/cfd/src/Containers/dynamic-leverage/dynamic-leverage-modal-content.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { dynamicLeverages } from '../../Constants/dynamic-leverage-content/dynamic-leverage-content';
+import { DynamicLeverageMarketCard } from './dynamic-leverage-market-card';
+
+const DynamicLeverageModalContent = () => {
+    return (
+        <div className='dynamic-leverage-modal'>
+            <Text data-testid='header-text' size='xs'>
+                {localize(`
```

Comment:
let's not use backticks and pass string instead. 
You can refer to the readme file on this.